### PR TITLE
fix rendering children as zero number

### PR DESCRIPTION
**Pull request by JiLiZART**
React allows pass `0` (number, not string) as children prop and properly render it
React sandbox example
https://codesandbox.io/s/cranky-dust-7tjqw

### CODE REVIEW

File: compat/src/Children.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { toChildArray } from 'preact';
 
 const mapFn = (children, fn) => {
-	if (!children) return null;
+	if (typeof children === 'undefined' || children === null) return null;
```
Comment (JoviDeCroock) [inline compat/src/Children.js:4]:
```suggestion
	if (children === undefined || children === null) return null;
```

Comment (JiLiZART) [reply compat/src/Children.js:4]: I think this check more robust, since we can overwrite `undefined`variable.

File: compat/src/Children.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { toChildArray } from 'preact';
 
 const mapFn = (children, fn) => {
-	if (!children) return null;
+	if (children === undefined || children === null) return null;
```
Comment (JoviDeCroock) [inline compat/src/Children.js:4]: Found a shorter equivalent, thanks for notifying us about this bug!

```suggestion
	if (children == null) return null;
```

**Review by JoviDeCroock — APPROVED** [review]
Awesome, thank you so much! 💯

**Review by marvinhagemeister — APPROVED** [review]
Sweet, thank you so much for the PR 🙌

**Pull request merged**